### PR TITLE
Create ui-c8y-1021-4-1-fix-duplicate-navigation-entries-when-configuring-cockpit-root-nodes-7412-graft.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [MTM-61026](https://cumulocity.atlassian.net/browse/MTM-61026)
Original PR: [7584](https://github.com/Cumulocity-IoT/cumulocity-ui/pull/7584)

[MTM-61026]: https://cumulocity.atlassian.net/browse/MTM-61026?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1021-4-1-fix-duplicate-navigation-entries-when-configuring-cockpit-root-nodes-7412-graft.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Fix duplicate navigation entries when configuring cockpit root nodes (#7412) [GRAFT][release/cd] (#7584)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1021-4-1-fix-duplicate-navigation-entries-when-configuring-cockpit-root-nodes-7412-graft.md]: The Claude AI suggested to use the following title:
```suggestion
title: Fix duplicate navigation entries when configuring cockpit root nodes
```

File: content/change-logs/application-enablement/ui-c8y-1021-4-1-fix-duplicate-navigation-entries-when-configuring-cockpit-root-nodes-7412-graft.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-61026
+version: 1021.4.1
+---
+Fix duplicate navigation entries when configuring cockpit root nodes (#7412) [GRAFT][release/cd] (#7584)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1021-4-1-fix-duplicate-navigation-entries-when-configuring-cockpit-root-nodes-7412-graft.md]: The Claude AI suggested to use the following body:
```suggestion
In certain scenarios, duplicate navigation entries could appear in the Cumulocity cockpit when configuring root nodes. This issue has been resolved, ensuring that navigation entries are now displayed correctly without any duplicates. Cockpit users will no longer encounter redundant entries, leading to a cleaner and more intuitive navigation experience.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Comment by BeateRixen** [comment]
@DawidSAG Please check this PR

File: content/change-logs/application-enablement/ui-c8y-1021-4-1-fix-duplicate-navigation-entries-when-configuring-cockpit-root-nodes-7412-graft.md (hunk 1 above)
Comment (DawidSAG) [inline content/change-logs/application-enablement/ui-c8y-1021-4-1-fix-duplicate-navigation-entries-when-configuring-cockpit-root-nodes-7412-graft.md]:
```suggestion
title: Prevent duplicate entries in navigation when configuring top level nodes
```

File: content/change-logs/application-enablement/ui-c8y-1021-4-1-fix-duplicate-navigation-entries-when-configuring-cockpit-root-nodes-7412-graft.md (hunk 2 above)
Comment (DawidSAG) [inline content/change-logs/application-enablement/ui-c8y-1021-4-1-fix-duplicate-navigation-entries-when-configuring-cockpit-root-nodes-7412-graft.md]:
```suggestion
Fixed an issue where editing top level nodes in the "Application configuration" would create duplicate entries in the main navigation menu. The navigator now properly updates without creating redundant menu items.
```

**Review by DawidSAG — APPROVED** [review]

File: content/change-logs/application-enablement/ui-c8y-1021-4-1-fix-duplicate-navigation-entries-when-configuring-cockpit-root-nodes-7412-graft.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-61026
+version: 1021.4.1
+---
+Fixed an issue where editing top level nodes in the "Application configuration" would create duplicate entries in the main navigation menu. The navigator now properly updates without creating redundant menu items.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1021-4-1-fix-duplicate-navigation-entries-when-configuring-cockpit-root-nodes-7412-graft.md]: @DawidSAG This is not yet clear. I assume you are talking about assets. Can you please add this info?

Comment (DawidSAG) [reply content/change-logs/application-enablement/ui-c8y-1021-4-1-fix-duplicate-navigation-entries-when-configuring-cockpit-root-nodes-7412-graft.md]: What about:

```suggestion
Fixed an issue where editing top level nodes in the "Application configuration" would create duplicate entries of assets in the main navigation menu. The navigator now properly updates without creating redundant menu items.
```

**Review by rahulc8y — APPROVED** [review]

File: content/change-logs/application-enablement/ui-c8y-1021-4-1-fix-duplicate-navigation-entries-when-configuring-cockpit-root-nodes-7412-graft.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Prevent duplicate entries in navigation when configuring top level nodes
+product_area: Application enablement & solutions
+change_type:
+  - value: change-VSkj2iV9m
+    label: Fix
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1021-4-1-fix-duplicate-navigation-entries-when-configuring-cockpit-root-nodes-7412-graft.md]:
```suggestion
    label: Cockpit
```

File: content/change-logs/application-enablement/ui-c8y-1021-4-1-fix-duplicate-navigation-entries-when-configuring-cockpit-root-nodes-7412-graft.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Prevent duplicate entries in navigation when configuring top level nodes
+product_area: Application enablement & solutions
+change_type:
+  - value: change-VSkj2iV9m
+    label: Fix
+component:
+  - value: component-YbYJ3gLU_
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1021-4-1-fix-duplicate-navigation-entries-when-configuring-cockpit-root-nodes-7412-graft.md]:
```suggestion
  - value: component-YdSEScrEC
```

File: content/change-logs/application-enablement/ui-c8y-1021-4-1-fix-duplicate-navigation-entries-when-configuring-cockpit-root-nodes-7412-graft.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-61026
+version: 1021.4.1
+---
+Fixed an issue where editing top level nodes in the "Application configuration" would create duplicate entries of assets in the main navigation menu. The navigator now properly updates without creating redundant menu items.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1021-4-1-fix-duplicate-navigation-entries-when-configuring-cockpit-root-nodes-7412-graft.md]:
```suggestion
Fixed an issue where editing top level nodes in the **Application configuration** page created duplicate entries of assets in the navigator. Now the navigator updates properly without creating redundant menu items.
```

File: content/change-logs/application-enablement/ui-c8y-1021-4-1-fix-duplicate-navigation-entries-when-configuring-cockpit-root-nodes-7412-graft.md [hunk 7]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Prevent duplicate entries in navigation when configuring top level nodes
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1021-4-1-fix-duplicate-navigation-entries-when-configuring-cockpit-root-nodes-7412-graft.md]:
```suggestion
title: Prevented duplicate entries in the navigator when configuring top level nodes
```

File: content/change-logs/application-enablement/ui-c8y-1021-4-1-fix-duplicate-navigation-entries-when-configuring-cockpit-root-nodes-7412-graft.md [hunk 8]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Prevented duplicate entries in the navigator when configuring top level nodes
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1021-4-1-fix-duplicate-navigation-entries-when-configuring-cockpit-root-nodes-7412-graft.md]:
```suggestion
title: Prevented duplicate asset entries in the navigator when configuring top level nodes
```

File: content/change-logs/application-enablement/ui-c8y-1021-4-1-fix-duplicate-navigation-entries-when-configuring-cockpit-root-nodes-7412-graft.md [hunk 9]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YdSEScrEC
+    label: Cockpit
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-61026
+version: 1021.4.1
+---
+Fixed an issue where editing top level nodes in the **Application configuration** page created duplicate entries of assets in the navigator. Now the navigator updates properly without creating redundant menu items.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1021-4-1-fix-duplicate-navigation-entries-when-configuring-cockpit-root-nodes-7412-graft.md]:
```suggestion
Fixed an issue where editing top level nodes in the **Application configuration** page in the Cockpit application created duplicate entries of assets in the navigator. Now the navigator updates properly without creating redundant menu items.
```

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**